semgrep_output_v1.atd

```diff
@@ -439,7 +462,7 @@ type severity
 type cli_error = {
   (* exit code? *)
   code: int;
-  (* TODO: use a variant (Warning | Error | ...) *)
+  (* TODO: use error_severity *)
   level: string;
```

Comment:
Is there a reason to not do so in this PR?

Reply:
I am very careful with any change that effect the CLI output. The semgrep-core output, not scared, it's used only internally, but the CLI output, people outside r2c are relying on it so I go slow there and prefer separate PR doing just one thing.